src/Raven.Server/Json/Converters/SliceJsonConverter.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using Newtonsoft.Json;
+using Raven.Client.Json;
+using Raven.Client.Json.Converters;
+using Raven.Server.ServerWide.Context;
+using Sparrow;
+using Sparrow.Json;
+using Voron;
+
+namespace Raven.Server.Json.Converters
+{
+    internal sealed class SliceJsonConverter : RavenJsonConverter
+    {
+        public static readonly SliceJsonConverter Instance = new SliceJsonConverter();
+
+        public override void WriteJson(JsonWriter writer, object value, JsonSerializer serializer)
+        {
+            if (!(writer is BlittableJsonWriter blittableJsonWriter))
+            {
+                throw new SerializationException($"{nameof(BlittableJsonWriter)} must to be used for write {nameof(LazyStringValue)}");
+            }
+
+            if (!(value is Slice slice))
+            {
+                throw new SerializationException($"Try to write non {nameof(Slice)} value with {nameof(SliceJsonConverter)}");
+            }
+
+            var buffer = new byte[slice.Size];
+            slice.CopyTo(buffer);
+            var strValue = Convert.ToBase64String(buffer);
+
+            blittableJsonWriter.WriteValue(strValue);
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            if (!(reader is BlittableJsonReader blittableReader))
+            {
+                throw new SerializationException($"{nameof(BlittableJsonReader)} must to be used for convert to {nameof(BlittableJsonReaderObject)}");
+            }
+
+            if (blittableReader.Value == null)
+            {
+                return null;
+            }
+
+            if (!(blittableReader.Value is string strValue))
+            {
+                throw new SerializationException($"Try to read {nameof(Slice)} from non string value");
+            }
+
+            //TODO To create slice by JsonOperationContext
+            if (!(blittableReader.Context is DocumentsOperationContext context))
```

Comment:
Why do you need `DocumentsOperationContext` and not just `JsonOperationContext`?

Reply:
[`Slice.From(context.Allocator, buffer, ByteStringType.Immutable, out var slice);`](https://github.com/ravendb/ravendb/pull/7397/files/1ab8eb87a068abb6e93c176b114e45468a6c1b1b#diff-d9907d27cc6f398f4ccad4f9d7d9823bR62)

As I understand it, only the TransactionOperationContext hold the Allocator that I need to create Slice. 
I can create it by myself but in this case, I will need to dispose it.
For that, I need to bring the allocator from the outside, the reader in this case,
and release the memory after the replay is done (or after the command execute).
It seems to me very messy to hold two objects (context & allocator) than handle memory in the reader.
The other option I can think about is to hold an allocator in the JsonOperationContext but this is a huge change to the system. Maybe lazy allocator?
Or there is something I didn't think about...